### PR TITLE
solution

### PR DESCRIPTION
**Pull request by uzlss**
I forgot how to create short-in annotation for Distance, int, float and mate.academy/learn was dead and I have no clue how to google this request.... may be im just freaking out over....

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        if isinstance(other, (int, float)):
+            return Distance(
+                self.km * self.number_return(other)
+            )
+
+    def __truediv__(self, other: float | int) -> Distance:
+        if isinstance(other, (int, float)):
```
Comment (ArsenPidhoretskyi) [inline app/main.py:32]: This check is not needed
The same in mul
Because we already say that we expect only float or int

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+        return self.km == self.number_return(other)
+
+    def __le__(self, other: Distance | int | float) -> bool:
+        return self.km <= self.number_return(other)
+
+    def __ge__(self, other: Distance | int | float) -> bool:
+        return self.km >= self.number_return(other)
+
+    @staticmethod
+    def number_return(number: Any) -> int | float:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:53]: It also can return None
when if return false and elif return false

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Comment by ArsenPidhoretskyi** [comment]
You can do
OtherInputType = Union[float, int, Distance]
And then just write
other: OtherInputType

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+
+    @staticmethod
+    def number_return(number: Any) -> int | float | None:
+        if isinstance(number, Distance):
+            return number.km
+        elif isinstance(number, (int, float)):
+            return number
+
+
+OtherInputType = Union[float, int, Distance]
```
Comment (vasylhnatiuk) [inline app/main.py:59]: write it before Distance, not after

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by Vasyl-Smutok — APPROVED** [review]
Good 👍

**Pull request closed without merging**